### PR TITLE
CORE-69 readme and contributing

### PR DESCRIPTION
**Pull request by cottand**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Contributing to ivann
+
+First of all, thanks for taking the time to contribute!
+
+Ivann is currently maintained by four
```
Comment (tomproberts) [inline CONTRIBUTING.md]: Aren't there 7 of us, lol

File: README.md [hunk 2]
```diff
@@ -6,30 +6,31 @@
...
-## Project setup
-```
-npm install
-```
+## What can I make with ivann?
 
-### Compiles and hot-reloads for development
-```
-npm run serve
+Ivann was engineered to build neural networks. You will find
```
Comment (tomproberts) [inline README.md]: "Ivann **is** engineered" would sound better IMO

**Review by KacperKazan — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -6,30 +6,31 @@
...
+we put extra effort in building your typical CNN.
+
+But we also put a lot of effort in basically supporting all
+of PyTorch, and we allow for 'inlining' python code in an otherwise fully
+UI-oriented developing experience. So it is up to you: you could stick
```
Comment (tomproberts) [inline README.md]: "you could stick using..." -> "you can either stick to using..."

**Review by tomproberts — CHANGES_REQUESTED** [review]
Nice stuff, just those 3 things

File: CONTRIBUTING.md (hunk 1 above)
Comment (cottand) [reply CONTRIBUTING.md]: Omfg habit

I am _so_ sorry

**Review by tomproberts — DISMISSED** [review]
Awesome stuff! 🎉

**Review by tomproberts — DISMISSED** [review]
Coolio

**Review by KacperKazan — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Contributing to ivann
+
+First of all, thanks for taking the time to contribute!
+
+Ivann is currently maintained by seven
+[Imperial College](https://www.imperial.ac.uk/computing) students.
+We will work hard to keep ivann alive and will try to tend to
```
Comment (tomproberts) [inline CONTRIBUTING.md]: "and will try to address issues and PRs..."

**Review by lancelotblanchard — DISMISSED** [review]
Stupendous work!

**Review by tinasandu — APPROVED** [review]

**Review by KacperKazan — APPROVED** [review]

**Review by tomproberts — APPROVED** [review]
Super Top!

**Review by gidonkatten — APPROVED** [review]

**Pull request merged**